django/db/backends/oracle/introspection.py

```diff
@@ -22,15 +23,11 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
         cx_Oracle.TIMESTAMP: 'DateTimeField',
     }
 
-    try:
+    with suppress(AttributeError):
         data_types_reverse[cx_Oracle.NATIVE_FLOAT] = 'FloatField'
```

Comment:
Good. I've moved the data type into the map instead. There's a remark about `Unicode` in http://cx-oracle.readthedocs.io/en/latest/releasenotes.html?highlight=unicode (I won't change anything here, but probably those data types should be in the reverse map as well)?